### DISCUSSION TITLE
Fresh install fails to build

### DISCUSSION THREAD

**Discussion by cfjedimaster** [adonisjs/core#4178] [General]
This may be a bug and I'll file there if so, but wanted to check. I installed my first Adonis app following the directions here https://docs.adonisjs.com/guides/installation. After confirming it ran, and without touching any file at all, I stopped the server, ran `node ace build --production`, and got an error:

```
config/shield.ts:8:1 - error TS6133: 'Env' is declared but its value is never read.
```

This is easy to get around, just add `ignore-ts-errors`, but shouldn't a fresh, not touched install be able to build?

**Comment by Julien-R44** [comment]
No big deal, it's just a warning that there's an unused variable, nothing more

I've released a new version of @adonisjs/shield which solves the problem for future fresh applications. Thanks for noticing
